bin.js

```diff
@@ -121,27 +122,48 @@ yargs.command('run <system> [location] [test]', 'Run your package\'s tests in re
     // Add the raw package.json template
     const rnPkg = require(`${root}/template-package.json`)
     await writeFile(`${root}/package.json`, JSON.stringify(rnPkg, null, 2) + '\n')
-    try {
-      // Install any dependencies of the wrapper app
-      await logExec('npm', ['i'], { cwd: root, quiet: !verbose })
-    } finally {
-      // Add the package name to dependencies so it will compile
-      rnPkg.dependencies[packageJSON.name] = ''
-      await writeFile(`${root}/package.json`, JSON.stringify(rnPkg, null, 2) + '\n')
-    }
 
-    // Make sure the folder exists for copying over the package
-    console.log(`## react-native:preparing-dep [target=${target}]`)
-    await mkdirp(path.dirname(target))
+    // Install any dependencies of the wrapper app
+    await logExec('npm', ['i'], { cwd: root, quiet: !verbose })
+
+    console.log('## react-native:npm pre-pack')
+
+    // Merge devDependencies with dependencies
+    const { dependencies = {}, devDependencies = {} } = packageJSON
```

Comment:
I don't think its a good idea for `rn-tape` to prepare the package, it might be better to receive a package via pipe or file path

Reply:
What are the main disadvantages of having rn-tape do the packaging?

I originally went this way to reduce the amount of stuff a project author would need to do get started with rn-tape

Reply:
The disadvantage is immediately visible in get-random-values-polypony ... it builds it not from the main package but from a package that is built with a built script. the `npm pack` command would result in the wrong package - not to be tested.

Reply:
I'm not sure I follow. With get-random-values-polypony we specify the dist folder, right? Then that would be the folder that `npm pack` runs on, so it wouldn't bother with the top level.

Reply:
You are absolutely right! `rn-tape` just needs to prepare the dist folder first. Sorry for the distraction.